### PR TITLE
The AIIDALAB_DEFAULT_APPS variable is whitespace delimited.

### PR DESCRIPTION
**Pull request by csadorf**
Specifying newline-delimited variables is typically not well supported
in various contexts and iterating over a whitespace delimited variable
is also easier.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,14 +11,17 @@ ENV AIIDALAB_APPS ${AIIDALAB_HOME}/apps
...
-# Example for a properly formatted AIIDALAB_DEFAULT_APPS variable:
-#   aiidalab-widgets-base@https://github.com/aiidalab/aiidalab-widgets-base@v1.0
-#   quantum-espresso@https://github.com/aiidalab/aiidalab-qe@v1.1
+#
+# Example for setting the AIIDALAB_DEFAULT_APPS variable:
+#
+#   AIIDALAB_DEFAULT_APPS+="aiidalab-widgets-base@https://github.com/aiidalab/aiidalab-widgets-base@v1.0 "
+#   AIIDALAB_DEFAULT_APPS+="quantum-espresso@https://github.com/aiidalab/aiidalab-qe@v1.1 "
+#
+# Please not the space at the end to ensure that entries are whitespace delimited.
```
Comment (yakutovicha) [inline Dockerfile:24]:
```suggestion
# Please do not put space at the end to ensure that entries are whitespace delimited.
```
unless I got it wrong.

**Review by yakutovicha — COMMENTED** [review]
Thanks, @csadorf, only a small comment.

File: Dockerfile (hunk 1 above)
Comment (csadorf) [reply Dockerfile:24]: It should be:
```suggestion
# Please note the space at the end to ensure that entries are whitespace delimited.
```
However, I found placing the whitespace at the beginning is actually even better. I'll push the change.

**Review by yakutovicha — APPROVED** [review]
Looks good! Thanks, @csadorf!

**Pull request merged**